data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -0,0 +1,140 @@
+{
+  "version": 1,
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "_source": {
+        "enabled": true
+      },
+      "properties": {
+        "droppedEventsCount": {
+          "type": "integer"
+        },
+        "attributes": {
+          "type": "object"
```

Comment:
This is the place, where component templates come in handy, that model semantic conventions on any of the attributes objects. This should not be blocked by this choice, is it?